src/components/common/ui/form/dropdown-mui/Dropdown.tsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable react/prop-types */
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { ChevronDownIcon } from '@heroicons/react/24/solid';
+import { Box, Popper, Typography } from '@mui/material';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { useField } from 'formik';
+
+import { FieldState } from '@/components/common/ui/form/common/types';
+
+import { dropdown, input, popper, remark } from './Dropdown.styles';
+
+interface DropDownOption {
+  label: string;
+  id: string;
+}
+interface DropdownProps {
+  options: DropDownOption[];
+  label?: string;
+  name?: string;
+  isDisabled?: boolean;
+  placeholder?: string;
+  numberOfOptions?: number;
+  isSuccessOnDefault?: boolean;
+  defaultRemark?: string;
+  showRemark?: boolean;
+  size?: DropDownSize;
+  noOptionsText?: string;
+  onChange?: () => void;
```

Comment:
why do we need an `onChange`? where will it be used?

Reply:
copied props from the previous dropdown, idk if it's needed, should I remove it?